frontend/src/core/editor/global/views/editorView.js

```diff
@@ -45,14 +46,14 @@ define(function(require){
       Origin.editor.pasteParentModel = false;
       Origin.editor.isPreviewPending = false;
       this.currentCourseId = Origin.editor.data.course.get('_id');
-
+      this.currentCourse = Origin.editor.data.course;
```

Comment:
I'll give this PR a +1, but I wonder whether it's safe to keep hold of the current course object like this. I assume it is, but would appreciate a more experienced authoring tool developer confirming.